ontology/uco-gufo.ttl

```diff
@@ -151,6 +152,18 @@ uco-action:startTime
 		;
 	.
 
+uco-analysis:AnalyticResult
+	a gufo:SubKind ;
+	.
```

Comment:
Since `uco-analysis:AnalyticResult` is typed as a `gufo:SubKind` is should also be a superclass of the ultimate sortal, i.e., a `gufo:Kind` where it gets its identitity criterion from. I seem to miss that here.

Reply:
Ah, sorry, it was a bit obscured by me not repeating triples from UCO.  `analysis:AnalyticResult` is a direct subclass of `core:Assertion`:

https://github.com/ucoProject/UCO/blob/1.3.0/ontology/uco/analysis/analysis.ttl#L34-L39